src/App.js

```diff
@@ -27,14 +20,8 @@ function App() {
           <Routes>
             <Route exact path="/" element={<Main />} />
             <Route path="/comments" element={<Comments />} />
-            <Route path="/search" element={<SearchPage />} />
-            <Route path="/sign-up" element={<Comments />} />
-            <Route path="/movie" element={<MoviesTab />} />
-            <Route path="/show" element={<TvShowsTab />} />
-            <Route path="/developers" element={<DevTeam />} />
-            <Route path="/showtimes" element={<ShowtimesTab />} />
-            <Route path="/dashboard" element={<Dashboard />} />
-            <Route path="/podcast" element={<PodCast/>}/>
```

Comment:
no sure if deleting all of these pages is going to make the app work properly

Reply:
also for these